docs/eln/dev_docker_install.mdx

```diff
@@ -142,6 +142,10 @@ The development mode setup is straight forward and requires only a few steps. Th
 ```
 cd ~
 git clone https://github.com/ptrxyz/chemotion.git
+git remote update
+git fetch --all
+git pull --all
```

Comment:
I think `fetch` and `pull` are redundant. `pull` does a `fetch` before merging branches.

Reply:
git fetch tells your local git to retrieve the latest meta-data from the remote (yet doesn’t do any file transferring. It’s more like just checking to see if there are any changes available).

git pull on the other hand does that AND brings (copy) of those changes from the remote repository.

Should we then only keep "git pull --all" and ignore "git fetch --all"

Reply:
> Should we then only keep "git pull --all" and ignore "git fetch --all" ?

Sounds good :+1: